src/bootstrap/config/tests.rs

```diff
@@ -19,6 +19,8 @@ fn download_ci_llvm() {
     assert_eq!(parse_llvm(""), if_available);
     assert_eq!(parse_llvm("rust.channel = \"dev\""), if_available);
     assert!(!parse_llvm("rust.channel = \"stable\""));
+    assert!(parse_llvm("llvm.assertions = true \r\n build.target = [\"x86_64-unknown-linux-gnu\"] \r\n llvm.download-ci-llvm = \"if-available\""));
+    assert_eq!(parse_llvm("llvm.assertions = true \r\n build.target = [\"aarch64-apple-darwin\"] \r\n llvm.download-ci-llvm = \"if-available\""), false);
```

Comment:
Seems specifying target in config under test is not affecting `config.build.triple`, it is still `"aarch64-apple-darwin"` as my machine.
> ./x.py test src/bootstrap/ --target x86_64-unknown-linux-gnu --test-args download_ci_llvm
is not helping either: `x.py` complains about argument.

Reply:
you want `build.build`, not `build.target`. https://github.com/rust-lang/rust/blob/a377893da2cd7124e5a18c7116cbb70e16dd5541/config.toml.example#L188-L194